### PR TITLE
Inital Commit Ganesh for Create job Page

### PR DESCRIPTION
**Pull request by gmganesh3110**

### CODE REVIEW

File: layouts/default.vue [hunk 1]
```diff
@@ -6,7 +6,8 @@ const styleStore = useStyleStore();
 <template>
   <div :class="{ dark: styleStore.darkMode }">
     <div class="bg-gray-50 dark:bg-slate-800 dark:text-slate-100">
-      <slot />
+      <FormsCreateJob />
+      <!-- <slot /> -->
```
Comment (panthsolanki) [inline layouts/default.vue]: don't add form here, create page under career folder

File: pages/careers/CreateJob.vue [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+            <select class="mt-3 rounded-md h-12" v-model="jobData.jobType">
+              <option>Full Time</option>
+              <option>Part Time</option>
+              <option>Freelance</option>
+            </select>
+          </div>
+        </FormsFormField>
+      </div>
+      <div class="w-full flex justify-center">
+        <button
```
Comment (panthsolanki) [inline pages/careers/CreateJob.vue]: use BaseButtons component

File: layouts/default.vue [hunk 3]
```diff
@@ -9,4 +9,4 @@ const styleStore = useStyleStore();
       <slot />
     </div>
   </div>
-</template>
```
Comment (panthsolanki) [inline layouts/default.vue]: dont make any change in this file

File: components/Forms/CreateJob.vue [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+      </div>
+      <div class="lg:col-span-6">
+        <div class="ltr:text-left rtl:text-right">
+          <label for="name" class="font-semibold">Job Type</label>
+          <br />
+          <div class="checkboxes">
+            <div>
+              <input
+                class="checkboxinput"
+                type="checkbox"
```
Comment (panthsolanki) [inline components/Forms/CreateJob.vue]: checkbox should be in one single group, pls update the logic

File: components/Forms/CreateJob.vue [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+              <label for="" class="font-semibold">Work From Home</label>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="lg:col-span-6">
+        <div class="ltr:text-left rtl:text-right">
+          <label for="name" class="font-semibold">Education Requirments</label>
+          <br />
+          <div class="checkboxes">
```
Comment (panthsolanki) [inline components/Forms/CreateJob.vue]: checkbox should also be in single group

**Review by panthsolanki — CHANGES_REQUESTED** [review]

File: layouts/default.vue (hunk 1 above)
Comment (gmganesh3110) [reply layouts/default.vue]: Removed SIr

File: pages/careers/CreateJob.vue (hunk 2 above)
Comment (gmganesh3110) [reply pages/careers/CreateJob.vue]: Used Base Buttons and Updated sir

**Review by panthsolanki — DISMISSED** [review]

**Review by panthsolanki — APPROVED** [review]

**Pull request merged**